tests/spread/tools/snapcraft-yaml.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -e
+
+set-base()
```

Comment:
Would you mind using underscores here, please? Hyphens [are non-standard](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_235), and don't follow the convention already established in these scripts.

Reply:
I just dislike underscores, but very well